Automattic/woocommerce-payments
# Remove "Set-up refund policy" Inbox note

Fixes #7776

#### Changes proposed in this Pull Request

 - Removes "Set-up returns & refund policy" note .

See: pdqJU4-3q4-p2

 - Update Psalm configuration to let Psalm continue if it cannot access parts of `docker` directory, which is completely normal due to uids mess between container and host.

#### Testing instructions

 - Set-up a new WooCommerce store with WooPayments
 - Onboard WooPayments
 - Ensure that Inbox doesn't contain a note titled "Set-up refund policy"

-------------------


**Post merge**
